### PR TITLE
enhancements in client.sh

### PR DESCRIPTION
**Pull request by d78ui98**
Made `client.sh` less complex, readable and added usage function

### CODE REVIEW

**Comment by d78ui98** [comment]
I have also tested this one multiple times.

**Comment by dashohoxha** [comment]
The same comments from the other branch and pull request apply on this one too.

File: client.sh [hunk 1]
```diff
@@ -1,47 +1,21 @@
 #!/bin/bash
-#### This is a simple script that creates LTSP clients 
-#### These clients are diskless and only boot from network
-#### So make sure LTSP server is running before executing script
+#################################################
+#  Simple script that creates LTSP clients      #
+#  These clients can only boot from network     #
+#  LTSP server is running before running script	#
```
Comment (akash0x53) [inline client.sh:5]: just remove white space and correct alignment so it will not look weird.

**Review by akash0x53 — CHANGES_REQUESTED** [review]

File: client.sh (hunk 1 above)
Comment (d78ui98) [reply client.sh:5]: I dont know how but, it only looks weird here.
if you check here - https://github.com/docker-scripts/dev--LTSP/blob/enhance/client.sh
or my editor you wont see the white space.

**Review by akash0x53 — APPROVED** [review]

**Pull request merged**